### PR TITLE
Adds info about drag-n-drop for CC

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2878](https://openedx.atlassian.net/browse/DOC-2878)

You can play with this on stage, see the draft release notes (with images) in https://github.com/edx/edx-documentation/pull/984, and read Chris's info in https://github.com/edx/edx-platform/pull/11438
### Date Needed

Releasing 20 Apr 2016
### Reviewers
- [x] Subject matter expert: @clrux 
- [ ] Subject matter expert: 
- [x] Doc team review (copy edit): @srpearce 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/glossary/glossary.rst [hunk 1]
```diff
@@ -128,7 +128,11 @@ C
 
 **Closed Captions**
 
-  See :ref:`Transcript<T>`.
+  A transcript of the audio portion of a video file that is overlaid on the
```
Comment (downzer0) [inline en_us/shared/glossary/glossary.rst]: It's really the current spoken part of the transcript. Is that what you mean by "audio portion"?

File: en_us/shared/students/SFD_video_player.rst [hunk 2]
```diff
@@ -47,8 +47,9 @@ by an explanation of each option or control on the video player.
    your keyboard or select this control again.
 
 8. **Closed caption**: You can show an overlaid transcript of the audio portion
-   of the file by selecting this control. Select this control again to hide the
-   captions.
+   of the file by selecting this control. If you show the captions, you can
+   move them to different areas on the video screen by dragging and dropping
+   them. To hide the captions, select this control again.
```
Comment (srpearce) [inline en_us/shared/students/SFD_video_player.rst:52]: Nit: Maybe shorten this in the same way as above ("If you show the captions, you can drag them to different areas on the video screen").

File: en_us/shared/glossary/glossary.rst [hunk 3]
```diff
@@ -128,7 +128,11 @@ C
 
 **Closed Captions**
 
-  See :ref:`Transcript<T>`.
+  The spoken part of the transcript for a video file, which is overlaid on the
+  video as it plays. Closed captions can be moved to different areas on the
+  video screen by dragging and dropping.
```
Comment (srpearce) [inline en_us/shared/glossary/glossary.rst]: Nit: I would make this active and a little shorter ("You can drag closed captions to different areas on the video screen").

**Comment by srpearce** [comment]
Two nits, but 👍  overall

**Pull request merged**